src/index.js

```diff
@@ -5,107 +5,209 @@ import './view/css/style.css';
+}
 
-  const setName = (newName) => {
-    this.name = newName;
-  }
+// remove project from projectList, save local and render
+function removeProject(project) {
+  projectList.splice(projectList.indexOf(project), 1);
+  alert(`${project.title} deleted`); // eslint-disable-line no-alert
 
-  const getName = () => {
-    return this.name;
-  }
+  saveLocal();
+  location.reload(); // eslint-disable-line no-restricted-globals
+}
 
-  return {
-    addTask,
-    deleteTask,
-    setName,
-    getName,
-    taskArr
-  };
+function removeTodo(project, todo) {
+  project.todos.splice(project.todos.indexOf(todo), 1);
+  alert(`${todo.title} deleted`); // eslint-disable-line no-alert
 
+  saveLocal();
+  location.reload(); // eslint-disable-line no-restricted-globals
 }
 
-const task = (tName, tDescription, tDate, tChecked) => {
-  const name = tName;
-  const description = tDescription;
-  const date = tDate;
-  const checked = tChecked;
-
-  const setName = (newName) => {
-    this.name = newName;
-  }
-
-  const getName = () => {
-    return this.name;
-  }
-
-  const setDescription = (newDescription) => {
-    this.description = newDescription;
-  }
-
-  const getDescription = () => {
-    return this.description;
-  }
-
-  const setDate = (newDate) => {
-    this.date = newDate;
-  }
-
-  const getDate = () => {
-    return this.date;
-  }
-
-  const changeChecked = () => {
-    this.checked = !this.checked;
-  }
-
-  const getChecked = () => {
-    return this.checked;
-  }
-
-  return {
-    setName,
-    setDescription,
-    setDate,
-    changeChecked,
-    getName,
-    getDescription,
-    getDate,
-    getChecked
-  };
+function editTodo(todoIndex) {
+  const newName = document.getElementById('edit-todo-title').value;
+  projectList[currentProjectIndex].todos[todoIndex].title = newName;
+
+  const newDate = document.getElementById('edit-todo-dueDate').value;
+  projectList[currentProjectIndex].todos[todoIndex].dueDate = newDate;
+
+  const newPrior = document.getElementById('edit-todo-priority').value;
+  projectList[currentProjectIndex].todos[todoIndex].priority = newPrior;
+
+  const newDesc = document.getElementById('edit-todo-description').value;
+  projectList[currentProjectIndex].todos[todoIndex].description = newDesc;
+
+  saveLocal();
+  renderTodos(projectList[currentProjectIndex]); // eslint-disable-line no-use-before-define
 }
 
-const render = () => {
+function setTodoModal(todoIndex) {
+  document.getElementById('edit-todo-title').value = projectList[currentProjectIndex].todos[todoIndex].title;
+  document.getElementById('edit-todo-dueDate').value = projectList[currentProjectIndex].todos[todoIndex].dueDate;
+  document.getElementById('edit-todo-priority').value = projectList[currentProjectIndex].todos[todoIndex].priority;
+  document.getElementById('edit-todo-description').value = projectList[currentProjectIndex].todos[todoIndex].description;
+}
 
+// add todo to project save local and render
+function addTodo(todoList) {
```

Comment:
You ought to use ES6 Syntax for this project:
Which means usage of arrow functions instead of the function keyword.
See this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/es6_requirement.png) from pathwright